modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/notification/DDMFormEmailNotificationSender.java

```diff
@@ -424,13 +424,12 @@ protected String getViewFormEntriesURL(
 			DDMFormInstance ddmFormInstance, ThemeDisplay themeDisplay)
 		throws PortalException {
 
-		Group group = _groupLocalService.getGroup(ddmFormInstance.getGroupId());
-
 		String portletNamespace = _portal.getPortletNamespace(
 			DDMPortletKeys.DYNAMIC_DATA_MAPPING_FORM_ADMIN);
 
 		return _portal.getSiteAdminURL(
-			themeDisplay.getPortalURL(), group,
+			themeDisplay.getPortalURL(),
+			_groupLocalService.getGroup(ddmFormInstance.getGroupId()),
```

Comment:
You really got the BChan SF spirit, @carolmariaabb! 😄 👏

Reply:
😝